test/runnable/previewin.d

```diff
@@ -155,25 +154,26 @@ struct WithDtor
 @safe pure nothrow @nogc:
 
 // By value
-void testin1(in uint p) {}
+void testin1(in uint p) { static assert(!__traits(isRef, p)); }
 // By ref because of size
-void testin2(in ulong[4] p) {}
+void testin2(in ulong[64] p) { static assert(__traits(isRef, p)); }
```

Comment:
Increasing the size just to be on the safe side - some ABIs might be able to pass a 32-bytes aggregate in registers.

Reply:
Doesn't seem wise to add a static assert here.  You're forcing a behaviour when there is no spec on one.

Reply:
It's not spec'd, but if a 512-bytes arg is ever to be copied, then `-preview=in` isn't implemented as intended.

Reply:
Do you mean, passed in memory?

Reply:
I mean passed by-value, i.e., the thing that is tested that it's not.

Reply:
Well, isRef would mean that no copy is made?  There would still need to be spec on that though.  From my reading of it, only non-trivial types must be passed by ref.  Everything else either explicitly by value, or unenforceable.

Reply:
It's not spec'd, but do you see any ABI on the horizon where it would be advantageous to pass half a KB by value over passing a ref? If that's the case in a few decades, the test can be adapted, until then, it makes sure the implementation is up to the expectation of eliding expensive copies.